portal-impl/src/com/liferay/portal/upgrade/v7_4_x/UpgradeRelease.java

```diff
@@ -46,7 +46,6 @@ protected void doUpgrade() throws Exception {
 		"com.liferay.document.library.service",
 		"com.liferay.document.library.video",
 		"com.liferay.dynamic.data.mapping.service",
-		"com.liferay.frontend.js.a11y.web",
```

Comment:
@thiago-buarqque should we manually be removing this?

Reply:
Discussed here: https://liferay.slack.com/archives/C08KJGY3DEW